docs/provisioners/helm.md

```diff
@@ -166,18 +149,18 @@ EOF
 bundledeployment.core.rukpak.io/my-ahoy created
 ```
 
-Next, check the Bundle status via:
+Next, check the BundleDeployment status via:
 
 ```bash
-kubectl get bundle -l app=my-ahoy
+kubectl get bundledeployment -l app=my-ahoy
 ```
 
-Eventually the Bundle should show up as Unpacked:
+Eventually the BundleDeployment should show up as Unpacked:
 
 ```console
-$ kubectl get bundle -l app=my-ahoy
-NAME                 TYPE   PHASE      AGE
-my-ahoy-5764594dc8   git    Unpacked   33s
```

Comment:
The bundle deployment column output doesn't have a PHASE column, right? And I would assume that basically all BDs would move on from "Unpacked" to either a successful or failed installation fairly quickly. So this area might need a bit more tweaking.

Reply:
My bad, that's right. It only has install state. Tweaked this part a bit for all 3 provisioners.